### PR TITLE
Add support to get default flag value, and display it on the override…

### PR DESCRIPTION
**Pull request by christor**
Add support to get default flag value, and display it on the overrides form.

The override form is really handy, but it would be even better to know what the value of the default is here.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
 node_modules
```
Comment (christor) [inline .gitignore]: This file is, obviously, less critical...feel free to ignore the changes to ignore :-)

Comment (mjt01) [inline .gitignore]: Happy for you to tidy this up and add it to your PR. By tidy up, maybe just remove `nbproject` as that looks unrelated.

File: dist/featureFlags.js [hunk 2]
```diff
@@ -64,7 +65,7 @@ angular.module('feature-flags').directive('featureFlagOverrides', ['featureFlags
                   '        <div class="feature-flags-name">{{flag.name || flag.key}}</div>'+
                   '        <div class="feature-flags-switch" ng-click="enable(flag)" ng-class="{\'active\': isOverridden(flag.key) && isOn(flag.key)}">ON</div>'+
                   '        <div class="feature-flags-switch" ng-click="disable(flag)" ng-class="{\'active\': isOverridden(flag.key) && !isOn(flag.key)}">OFF</div>'+
-                  '        <div class="feature-flags-switch" ng-click="reset(flag)" ng-class="{\'active\': !isOverridden(flag.key)}">DEFAULT</div>'+
+                  '        <div class="feature-flags-switch" ng-click="reset(flag)" ng-class="{\'active\': !isOverridden(flag.key)}">DEFAULT ({{isOnByDefault(flag.key)?\'ON\':\'OFF\'}})</div>'+
```
Comment (mjt01) [inline dist/featureFlags.js]: For readability I would prefer you have spaces around your ternary operators:
```({{ isOnByDefault(flag.key) ? \'ON\' : \'OFF\' }})```

File: src/featureFlagOverrides.directive.js [hunk 3]
```diff
@@ -16,7 +17,7 @@ angular.module('feature-flags').directive('featureFlagOverrides', function(featu
                   '        <div class="feature-flags-name">{{flag.name || flag.key}}</div>'+
                   '        <div class="feature-flags-switch" ng-click="enable(flag)" ng-class="{\'active\': isOverridden(flag.key) && isOn(flag.key)}">ON</div>'+
                   '        <div class="feature-flags-switch" ng-click="disable(flag)" ng-class="{\'active\': isOverridden(flag.key) && !isOn(flag.key)}">OFF</div>'+
-                  '        <div class="feature-flags-switch" ng-click="reset(flag)" ng-class="{\'active\': !isOverridden(flag.key)}">DEFAULT</div>'+
+                  '        <div class="feature-flags-switch" ng-click="reset(flag)" ng-class="{\'active\': !isOverridden(flag.key)}">DEFAULT ({{isOnByDefault(flag.key)?\'ON\':\'OFF\'}})</div>'+
```
Comment (mjt01) [inline src/featureFlagOverrides.directive.js]: For readability I would prefer you have spaces around your ternary operators:
```({{ isOnByDefault(flag.key) ? \'ON\' : \'OFF\' }})```

**Comment by mjt01** [comment]
Apologies i had some outstanding work around upgrading to eslint, that will be the cause of your merge conflicts.

**Comment by christor** [comment]
@mjt01 No problem. I've resolved the conflicts re-pushed
